mix.exs

```diff
@@ -43,16 +43,16 @@ defmodule Guardian.Mixfile do
 
   defp deps do
     [{:jose, "~> 1.8"},
-     {:phoenix, "~> 1.2.0", optional: true},
-     {:plug, "~> 1.0"},
+     {:phoenix, "~> 1.2.1", optional: true},
```

Comment:
@alexandrubagu could we update this to: `{:phoenix, "~> 1.2", optional: true}`?  I believe it'll address most of the 1.3rc1 concerns at the same time 👍